Input/input.html

```diff
@@ -0,0 +1,370 @@
+                  sandbox="allow-scripts allow-same-origin">
+                </iframe>
+              </section>
+
+            </div>
+
+          </section>
+
+          <hr>
+          <section>
+            <h3>Validating User Input for type="email"</h3>
+            <div class="validated-basic-input orange-bordered">
+              <label>Enter email ID:</label>
+              <br><br>
+              <input type="email" size="16" required class="custom custom-color"><span></span>
+            </div>
+
+            <div class="code-preview">
+              <h3>HTML Snippet</h3>
+              <section>
+                <iframe
+                  src="https://carbon.now.sh/embed?bg=rgba%280%2C0%2C0%2C1%29&t=night-owl&wt=none&l=htmlmixed&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=48px&ph=32px&ln=false&fl=1&fm=Hack&fs=13px&lh=133%25&si=false&es=2x&wm=false&code=%253Cinput%2520type%253D%2522email%2522%2520size%253D%252216%2522%2520required%2520class%253D%2522custom%2520custom-color%2522%253E%253Cspan%253E%253C%252Fspan%253E"
+                  style="width: 729px; height: 187px; border:0; transform: scale(1); overflow:hidden;"
+                  sandbox="allow-scripts allow-same-origin">
+                </iframe>
+              </section>
+
+            </div>
+
+          </section>
+
+          <hr>
+          <section>
+
+            <h3>Validating User Input for type="date"</h3>
+            <div class="validated-basic-input orange-bordered">
+              <label>Enter date:</label>
+              <br><br>
+              <input type="date" min="2021-01-01" max="2021-02-01" required class="custom custom-color"><span></span>
+            </div>
+
+            <div class="code-preview">
+              <h3>HTML Snippet</h3>
+              <section>
+                <iframe
+                  src="https://carbon.now.sh/embed?bg=rgba%280%2C0%2C0%2C1%29&t=night-owl&wt=none&l=htmlmixed&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=48px&ph=32px&ln=false&fl=1&fm=Hack&fs=13px&lh=133%25&si=false&es=2x&wm=false&code=%253Cinput%2520type%253D%2522date%2522%2520min%253D%25222021-01-01%2522%2520max%253D%25222021-02-01%2522%2520required%2520class%253D%2522custom%2520custom-color%2522%253E%253Cspan%253E%253C%252Fspan%253E"
+                  style="width: 909px; height: 187px; border:0; transform: scale(1); overflow:hidden;"
+                  sandbox="allow-scripts allow-same-origin">
+                </iframe>
+              </section>
+
+            </div>
+
+          </section>
+          <hr>
+          <section>
+            <h3>Checkout Page</h3>
+            <div class="user-checkout-details">
+              <div class="contact-information">
+                <h3>Contact information</h3>
+
+                <label>E-mail</label>
+                <div class="user-input">
+                  <i class="material-icons md-24">email</i><input id="email" required type="email"
+                    placeholder="Enter your email..."><span></span>
```

Comment:
The placeholder font size is too large. A little smaller font size for placeholder text and the input text would look better.